itemframework/src/item/abstract_item.cpp

```diff
@@ -325,6 +327,38 @@ ItemOutput* AbstractItem::addOutput(int type, QString const& description)
     return output;
 }
 
+void AbstractItem::removeInput(ItemInput* itemInput)
```

Comment:
Wow, did not spot that either! :sweat: 

We could just name it `remove(ItemOutput* output)` and overload for `ItemInput`, no need to make the names longer than they need to be.